### PR TITLE
Delete dead code

### PR DESCRIPTION
**Pull request by mondoreale**
In this PR I get rid of dead code. Less debris = less maintenance.

I try to manoeuvre in such a way that I don't remove DU-related code, although we're in a good spot to reshape things a bit. Either way, if anything gets lost, this here is a good place to scavenge some useful parts later on.

Cheers!

### CODE REVIEW

File: app/src/pages/AbstractStreamEditPage/AccessControlSection/index.tsx [hunk 1]
```diff
@@ -15,14 +15,14 @@ const AccessControlSection: React.FunctionComponent<Props> = ({ disabled: disabl
     const disabled = disabledProp || !canEdit
 
     return (
-        <TOCSection id="accessControl" title="Access control">
+        <Section title="Access control">
```
Comment (mondoreale) [inline app/src/pages/AbstractStreamEditPage/AccessControlSection/index.tsx:18]: There's more of this here. `TOCSection` (gone) got simplified and is now `Section`. Less moving parts.

Similar simplification happened in all other stream page `*Section` components.

**Review by tumppi — APPROVED** [review]
Goodbye 😭

**Comment by mondoreale** [comment]
I'll get that conflict resolved and merge in.

**Review by pmichalowski — APPROVED** [review]
My favourite type of PRs!

**Pull request merged**